### DISCUSSION TITLE
Does Ocelot support YAML files instead of JSON for Swagger configuration?

### DISCUSSION THREAD

**Discussion by allanchua101** [ThreeMammals/Ocelot#1904] [Q&A] [answered]
## Expected Behavior / New Feature

Hi, this is more of a question and not a bug, does Ocelot support YAML files instead of JSON for Swagger configuration?

**Comment by raman-m** [comment]
Ask Swagger team about YAML format!

**Comment by raman-m** [comment, marked as answer]
Officially we don't support and add Swagger to Ocelot core.
- [Not Supported | Swagger](https://ocelot.readthedocs.io/en/latest/introduction/notsupported.html#swagger)

But there is a good library that provides Swagger support:
- [MMLib.SwaggerForOcelot](https://github.com/Burgyn/MMLib.SwaggerForOcelot)
- PR #1429
